content/en/docs/specs/status.md

```diff
@@ -54,11 +54,12 @@ support for specific features can be found in the
 Note that, for each of the following sections, the **Collector** status is the
 same as the **Protocol** status.
 
-### [Tracing][]
+### Tracing
 
+- [Specification][tracing]
```

Comment:
Note: we shouldn't have links in headers, especially when the _entire_ header is a link, because that breaks the ToC entry. So I move the reference to the spec section into the body of the section. I did this change for each signal section.